build/build/src/paths.rs

```diff
@@ -42,17 +42,16 @@ ide_ci::define_env_var! {
 
 pub const EDITION_FILE_ARTIFACT_NAME: &str = "Edition File";
 
-pub const LIBRARIES_TO_TEST: [&str; 7] = [
-    "Examples_Tests",
-    "Geo_Tests",
-    "Image_Tests",
-    // Temporarily disabled due to https://www.pivotaltracker.com/story/show/184042416
-    // "Meta_Test_Suite_Tests",
-    "Table_Tests",
-    "Base_Tests",
-    "AWS_Tests",
-    "Visualization_Tests",
-];
+/// Get paths of directories containing standard library test project.
+pub fn discover_standard_library_tests(repo_root: &generated::RepoRoot) -> Result<Vec<PathBuf>> {
+    // The glob pattern will discover paths like "H:\NBO\enso\test\AWS_Tests\package.yaml".
+    let glob_pattern = "**/*_Tests/package.yaml";
```

Comment:
```suggestion
    let glob_pattern = "test/*_Tests/package.yaml";
```
What about the above?

I'm not sure if we need to or even want to search in other directories.

Reply:
There are plans to move tests to be next to each library being tested (i.e. something like `test` folder next to `src` of each Enso library), so this may need an update then as well.

Reply:
> What about the above?
> 
> I'm not sure if we need to or even want to search in other directories.

W attach the glob pattern to the `repo_root.test` which already paths to the `test` directory.

Reply:
> There are plans to move tests to be next to each library being tested (i.e. something like `test` folder next to `src` of each Enso library), so this may need an update then as well.

Yes. This should not be a problem.

Reply:
> > What about the above?
> > I'm not sure if we need to or even want to search in other directories.
> 
> We attach the glob pattern to the `repo_root.test` which already paths to the `test` directory.

In that case is the `**` even needed?

Unless I'm wrong in this context, usually `**` in a glob means multi-level directory traversal and I don't see why that would be needed - in our case we don't need nesting, all tests should be the immediate children of the `test` dir and looking 'deeper' seems like asking for trouble (if some test project contains an inner project for some reason).

That is a very minor nitpick though, as currently either way works 100% fine and we can always tweak it if needed. So feel free to ignore this suggestion.